lib/_stream_readable.js

```diff
@@ -678,6 +678,17 @@ Readable.prototype.on = function(ev, fn) {
 };
 Readable.prototype.addListener = Readable.prototype.on;
 
+Readable.prototype.removeListener = function(type, listener) {
+  Stream.prototype.removeListener.call(this, type, listener);
+
+  if (type === 'readable' && EE.listenerCount(this, type) === 0) {
+    var state = this._readableState;
```

Comment:
I will change it here and everywhere else in the file also; state variables, that is.